src/Pos/Util/Tx.hs

```diff
@@ -0,0 +1,142 @@
+-- | Pure functions for operations with transactions
+
+module Pos.Util.Tx
```

Comment:
From my point of `view` module `Util` should contain some general purpose functions which can be used everywhere, like some logging utility functions. All these functions are very specific. I think it's better to put into `Pos.Txp.Toil.Pure` but I don't remember quite well what is `Toil`. Maybe @gromakovsky can add something.